test/unit/server/ExpressHttpServer.test.ts

```diff
@@ -0,0 +1,82 @@
+import { ExpressHttpServer } from '../../../src/server/ExpressHttpServer';
+import { HttpHandler } from '../../../src/server/HttpHandler';
+import { HttpRequest } from '../../../src/server/HttpRequest';
+import { HttpResponse } from '../../../src/server/HttpResponse';
+import request from 'supertest';
+import { Server } from 'http';
+
+const handle = async(input: { request: HttpRequest; response: HttpResponse }): Promise<void> => {
+  input.response.writeHead(200);
+  input.response.end();
+};
+
+class SimpleHttpHandler extends HttpHandler {
+  public async canHandle(): Promise<void> {
+    return undefined;
+  }
+
+  public async handle(input: { request: HttpRequest; response: HttpResponse }): Promise<void> {
+    return handle(input);
```

Comment:
Linter doesn't hate this double use of `handle`?

Reply:
No since neither of them ever appears in the same namespace. But it's true that I didn't realize that this is not the clearest thing ever :D